src/include/indexlr-minimize.h

```diff
@@ -112,14 +130,51 @@ getMinimizers(const HashValues& hashes, const unsigned w)
 	return minimizers;
 }
 
-// Test the condition of a I/O stream.
-static inline void
-assert_good(const std::ios& stream, const std::string& path)
+static inline HashValues
+getMinimizers(const HashValues& hashes, const unsigned w, const BloomFilter& bloomFilter)
```

Comment:
There seems to be a lot of code in common between this and the other `getMinimizers`. Can you separate that functionality in another function?

Reply:
Removed with new implementation